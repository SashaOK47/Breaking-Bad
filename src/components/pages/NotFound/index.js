import style from "./styles.module.scss";
import { Link } from "react-router-dom";
const NotFound = () => {
  return (
    <div className={style.notFound}>
      <p className={style.message}>404 Not Found</p>
      <Link className={style.link} to="/">
        На главную
      </Link>
    </div>
  );
};
export default NotFound;
