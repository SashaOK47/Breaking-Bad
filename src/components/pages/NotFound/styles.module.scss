@import "../../../assets/styles/main.scss";
.notFound {
  flex: 1 1 auto;
  text-align: center;
  .message {
    margin: 5rem 0 2rem;
    font-size: 4rem;
    color: $colorError;
  }
  .link {
    text-decoration: underline;
    &:hover {
      text-decoration: none;
    }
  }
}
