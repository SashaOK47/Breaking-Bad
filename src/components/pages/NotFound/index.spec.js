import React from "react";
import NotFound from "./index";

describe("NotFound component", () => {
  it("should render NotFound component", () => {
    const component = shallow(<NotFound location="/" />);
    expect(component).toMatchSnapshot();
  });
});
