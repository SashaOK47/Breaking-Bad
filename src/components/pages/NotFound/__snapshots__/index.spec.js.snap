// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component should render NotFound component 1`] = `
<div
  className="notFound"
>
  <p
    className="message"
  >
    404 Not Found
  </p>
  <Link
    className="link"
    to="/"
  >
    На главную
  </Link>
</div>
`;
